examples/3d/elixir_euler_sedov_blast_wave_shockcapturing_amr.jl

```diff
@@ -0,0 +1,86 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations3D(1.4)
+
+initial_condition = Trixi.initial_condition_sedov_self_gravity
+boundary_conditions = Trixi.boundary_condition_sedov_self_gravity
```

Comment:
These should be exported (they are on `dev`).

Reply:
Fixed in 258c254.